testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -69,6 +74,13 @@ def test_dist_weights(u):
                                 [.707, -.707, 0, 0]]), 2)
 
 
+def test_distvalues_ag_universe(u, ag):
+    dist_universe = diffusionmap.DistanceMatrix(u, select='backbone').run()
+    dist_ag = diffusionmap.DistanceMatrix(ag).run()
+    assert_array_almost_equal(dist_universe.results.dist_matrix,
```

Comment:
For *new* tests we may as well use `assert_allclose` if the matrices contain floats per https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html

Reply:
Maybe we should consider replacing `assert_array_almost_equal` in other places as well as per the above link. I've updated it for only this test function.

Reply:
At this moment in time our approach is to only change tests we touch and make sure new tests use assert_allclose. Making large blanket changes makes it very hard to review PRs and also understand why changes were made based on history. If we need to do a blanket change we will do this in a specific PR in the future.